packages/gatsby/src/redux/actions/public.ts

```diff
@@ -1006,6 +1029,8 @@ actions.createNodeField = (
     node.fields = {}
   }
 
+  name = name || ``
```

Comment:
this feels like a functional difference.

Reply:
If `ICreateNodeInput` is correct then `name` can be `undefined`. Since it is used as argument below,  this would crash the program. I think that either this is needed or `name` is never `undefined`?